core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -356,6 +358,10 @@ protected void configure() {
 
     }
 
+    @SuppressWarnings({"EmptyMethod", "UnusedParameters"})
+    protected void containerIsCreated(String containerId) {
```

Comment:
It's actually `containerWasCreated`, or is it still in process?

Reply:
The creation is finished. I just named it in the style of the other callbacks `containerIsStarted` and `containerIsStarting`.

Reply:
Yes, was assuming something like this and makes sense to keep the style consistent. Just maybe less clear language.